website/content/configuration/k3s.md

```diff
@@ -3,7 +3,63 @@ title: Issues with K3s
 weight: 20
 ---
 
-K3S come with its own service load balancer named Klipper. You need to disable it in order to run MetalLB.
-To disable Klipper, run the server with the `--disable servicelb` option, as described in [K3s documentation](https://rancher.com/docs/k3s/latest/en/networking/
-)
+## Conflicts with K3s' own LoadBalancer implementation
 
+K3s come with its own service load balancer named Klipper. You need to disable it in order to run MetalLB.
+To disable Klipper, run the server with the `--disable servicelb` option, as described in [K3s documentation](https://rancher.com/docs/k3s/latest/en/networking/).
+
+As an alternative to disabling Klipper-LB you can configure MetalLB to only recognizes services that have been configured with a specific `loadBalancerClass` attribute,
+as described by the [installation documentation](https://metallb.universe.tf/installation/#setting-the-loadbalancer-class).
+
+For example, if you run the MetalLB `controller` and `speaker` pods with `--lb-class=metallb.universe.tf/metallb-class`, you can then directly reference this class in your services:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-metallb
+  annotations:
+    metallb.universe.tf/address-pool: ingress-ip-pool
+spec:
+  type: LoadBalancer
+  loadBalancerClass: metallb.universe.tf/metallb-class
+  ports:
+  - port: 443
+    targetPort: 8443
+```
+
+Klipper-LB will ignore this service due to its configured `loadBalancerClass`.
+
+## Exposing K3s' bundled ingress controller
```

Comment:
It's fine to keep this here for now, but I feel that having a section about ingress controllers which is independent of the cluster variant would be nice.

Reply:
I don't disagree. But I added it here because K3s specifically deploys Traefik with this toleration, which differs from the defaults of Traefik's native helm chart: https://github.com/k3s-io/k3s/blob/f099bfa508ede0532ede16ec6d4521cb8b144de6/manifests/traefik.yaml#L32